system/jlib/jmetrics.hpp

```diff
@@ -291,6 +290,45 @@ class jlib_decl HistogramMetric : public MetricBase
 };
 
 
+/*
+ * Enumerates the scaled time units
+ */
+enum TimeUnits
+{
+    HISTOGRAM_TIMEUNITS_NS,
+    HISTOGRAM_TIMEUNITS_US,
+    HISTOGRAM_TIMEUNITS_MS
+};
+
+class jlib_decl ScaledTimeFromCyclesHistogram : public MetricBase
```

Comment:
No need to restrict this to times.  It might be useful to aggregate memory as bytes but report as KB/MB.

Reply:
Times were added because it was specific to cycle_t. Since uint64 is the strict measurement type, the time units are no longer needed and have been removed.